docs/mobile-apps/virtual-usb.md

```diff
@@ -120,25 +118,33 @@ Run `java -jar virtual-usb-client --help` to see a full list of vUSB test config
   </TabItem>
   </Tabs>
 
-  If you'd like to use Sauce Connect Proxy, launch a tunnel in the Sauce Connect client, then add your `--tunnel-identifier` at the end of the above snippet. The vUSB client uses it to retrieve and secure test data. You can also set up a device proxy by adding proxy command options (see [Start Session](/dev/cli/virtual-usb/start-session)).
+  To use Sauce Connect Proxy: launch a tunnel in the Sauce Connect client, then add your [`--tunnel-identifier`](https://docs.saucelabs.com/dev/cli/virtual-usb/start-session#--tunnelidentifier), which the vUSB client will use to retrieve and secure test data. You can also set up a device proxy using [proxy command options](/dev/cli/virtual-usb/start-session).
 
   or
 
-  **Option 2: Connect to an existing live testing session**: Launch a live test on Sauce Labs (**Live** > **Mobile App** > Select Your App > **Choose Device** > Find Your Private Device > **Launch**). Then, you'll need to connect the session to your vUSB client. Open a new command line terminal and use the `sessions` command plus your credentials.
+  **Option 2: Connect to existing live testing session**: The second method is to start a live test directly on Sauce Labs, then use the command terminal to [connect that session](/dev/cli/virtual-usb/connect-session) that session to your vUSB client.
+
+  First, launch your test on Sauce Labs (**Live** > **Cross Browser** > **Mobile Real** > Find your Private Device > **Launch**). Next, locate your `--sessionId` by opening a new command line terminal and running the `sessions` command, along with your credentials.
```

Comment:
@kimsauce 

Is there a reason why `Find your Private Device` is not bold? The rest is

Reply:
good question! we only bold words that are UI elements. I remember the words "private device" appear here. So it's fair game to be bold 
![Screen Shot 2021-04-12 at 7 00 08 PM](https://user-images.githubusercontent.com/56411016/114485795-65921d00-9bc1-11eb-9cca-72c5e71bd39e.jpg)

Reply:
@kimsauce 
Can you also add this screenshot in the docs?

Reply:
I do have a similar one under here, which shows the Private Devices enabled -- https://docs.saucelabs.com/mobile-apps/virtual-usb/index.html#gather-credentials. Since I don't have access to Private Devices, this was a screenshot I took of your video. I was wondering, if you have the bandwidth, would you be able to take a screenshot of the same thing? And I will add to the doc.

Reply:
@kimsauce 

Is this what you need? 
![image](https://user-images.githubusercontent.com/11979740/114818008-49f26680-9dbb-11eb-97e9-bb742f7cd26a.png)

Reply:
hi wim, thanks for providing this! i was looking to grab a shot with the private devices toggle enabled, but Kev just granted me access to some private devices so I can take one and will add the arrows!

Reply:
Awesome, great that you now have access. If there's anything else I can do then please ping me